examples/gameroom/gameroom-app/src/scss/components/_toast.scss

```diff
@@ -18,7 +18,7 @@
   display: flex;
   flex-direction: row;
   align-items: center;
-  height: 3rem;
+  height: 4rem;
```

Comment:
These .scss seem unrelated to the fix

Reply:
It's to make the toast big enough to hold the error message, otherwise it was being clipped. The scroll change, in addition, is to ensure that the error message is not too big for the toast itself.

Reply:
I added a bit to the commit message to explain this change.